README.rst

```diff
@@ -22,43 +23,102 @@ Query Parameters:
 * ?[columnheader1]=[value1]&[columnheader3]=[value4]  (returns results that have both value1 and value2)
 
 
+Requirements
+------------
+
+In order to use autoapi, you'll need:
+
+* Python 3.5
+* npm
+
+Alternatively, if you use Docker for development and/or deployment, you don't
+need anything (except Docker, of course).
+
 
 Quickstart
 ----------
 
-::
+First, let's get dependencies sorted out::
+
+    pip3 install -r requirements.txt
+    npm install
+
+The easiest way to get started with autoapi is by serving CSV files from
+your local filesystem.
+
+Here's a quick way to generate some sample data::
+
+    mkdir data_sources
+    cat << EOF > data_sources/my_sample_data.csv
+    name,color
+    apple,red
+    grapefruit,pink
+    EOF
+
+Now you can add your sample data to a local SQLite database::
+
+    invoke apify "/data_sources/**.*"
+
+Finally, you can start the server with::
 
-    pip install invoke
-    invoke requirements
-    export AWS_ACCESS_KEY_ID="..."
-    export AWS_SECRET_ACCESS_KEY="..."
-    export AUTOAPI_BUCKET="..."
```

Comment:
I'm removing the S3 integration from the quick start section because setting up an S3 bucket, uploading a CSV to it, and configuring environment vars on the client doesn't feel like a "quick start" compared to using CSVs on the local filesystem :grin: